modules/Bio/EnsEMBL/Variation/Pipeline/BaseVariationProcess.pm

```diff
@@ -149,5 +149,28 @@ sub run_cmd {
   }
 }
 
+sub get_assembly {
```

Comment:
Bio::EnsEMBL::DBSQL::GenomeContainer::get_version() can be used to get assembly version